tools/cpp/windows_cc_configure.bzl

```diff
@@ -177,14 +177,14 @@ def _is_vs_2017(vc_path):
   return vc_path.find("2017") != -1
 
 
-def _find_vcvarsall_bat_script(repository_ctx, vc_path):
+def _find_vcvarsall_bat_script(repository_ctx, vc_path, check_exist = True):
```

Comment:
Please don't add `check_exists` here; instead, extract the error handling from this method to its call sites. Let this method just find the script (as its name says) or return None if not found, and let the caller deal with that (by either reporting it as an error or ignoring it).

Reply:
Make sense!